src/NumericalAlgorithms/LinearAlgebra/FindGeneralizedEigenvalues.hpp

```diff
@@ -0,0 +1,51 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+/// \file
+/// Defines function find_generalized_eigenvalues.
+
+#pragma once
+
+/// \cond
+class DataVector;
+class Matrix;
+namespace gsl {
+template <typename T>
+class not_null;
+} // namespace gsl
+/// \endcond
+
+/*!
+ * \ingroup NumericalAlgorithmsGroup
+ * \brief Solve the generalized eigenvalue problem for two matrices.
+ *
+ * This function uses the lapack routine dggev to solve the
```

Comment:
linking `dggev` to say the openblas documentation, or some other public blas documentation would be useful